### ISSUE TITLE
Import missing fields from "challenge responses" content

### ISSUE THREAD

**Issue by tanius** [edgeryders/discourse#28]
The initial migration from Drupal 7 to Discourse missed importing the "special" fields in content of type "challenge response". That means, all fields except title, body, author and creation date, especially:

* the three 140 character fields: "problem", "solution" etc.
* the three taxonomy fields: "collaborators", "multipliers" etc.

(Not urgent: there is no essential content in them, and we have stored the original Drupal node IDs for reference in Discourse so that adding content to the current import is possible anytime.)

**Comment by tanius** [comment]
Something to discuss with @Ladyniasan: Which of these special fields in challenge response content were really useful, and which were not? Background: adding fields to Discourse content is quite some effort, so we should avoid it as much as we can.

We will however import the content of the existing fields, by adding the image and the three 140-character summaries before the body text with our import script. The part to be avoided, if not essential, is adding special fields to the Discourse editor form. Can be done, but there might be better uses of the same developer time for the moment.

(@damingo : Please halt the implementation of this until we have an agreement here.)

**Comment by tanius** [comment]
Idea for a way to utilize custom fields (like for the Twitter-compatible summary) in Discourse without having to provide custom fields in the editor form:

The Discourse editor could have a button that inserts custom tags like `[twitter-summary] … [/twitter-summary]`. Upon saving, the content of these tags is copied into otherwise hidden custom post fields, which are possible in Discourse already (but difficult to show in the Editor form).

The same effect can be achieved by providing a topic templates (pre-filled text in the editor when starting a topic, can be configured in a category's settings). Then, the topic template would already contain all tags we want users to fill out, with some dummy content.

Anyway, users don't want to be forced to fill out form fields. The current Discourse interface feels nice, as it is "conversation like", allowing people to put in what they like. So maybe it's rather a job for an employee abroad to go over new content and edit in the Twitter summary etc..

**Comment by albertocottica** [comment]
Nice idea, but I would let go of this feature. As you say, people don't want to fill forms, and the site has a flow we do not want to break.

**Comment by tanius** [comment]
Agreed @albertocottica. Especially since nobody at all complained that the Twitter summaries etc. have not been imported so far.

With that change, it means we only want the "title image field" to be imported, by placing the corresponding picture at the beginning of the Discourse topics. Everything else is not relevant, but adding 300+ images to the site will make it a bit nicer to look at.
